src/perf.cc

```diff
@@ -28,7 +28,7 @@
 static long s_page_size = 0;
 
 struct perf_event_attr g_dd_native_attr = {
-    .size = sizeof(struct perf_event_attr),
+    .size = PERF_ATTR_SIZE_VER5,
```

Comment:
Tools like strace seem to process `perf_event_open()` better when we stick to the documented attribute size macro rather than use `sizeof()`